lib/rules/web/admin_console/4.1/route.xyaml

```diff
@@ -28,6 +43,12 @@ set_secure_route:
   action: uncheck_checkbox
   action: set_tls_termination
   action: set_insecure_traffic
+  action:
+    if_param:
+    - certificate_path
+    - private_key_path
+    - ca_certificate_path
+    ref: set_route_ca
```

Comment:
If we want to auto `re-encrypt` routes case, we will need one more file, `set_route_ca` maybe need updates again

Reply:
So how about using
```
set_secure_route:
  action:
    if_param: certificate_path
    ref: upload_certificate_file
  action: 
    if_param: private_key_path
    ref: upload_key_file
  action: 
    if_param: ca_certificate_path
    ref: upload_ca_certificate_file
  action: 
    if_param: destination_ca_cerficate
    ref: upload_destination_ca_crt_file 
```

Reply:
Ok, will update soon.